Odata-docs/odatalib/v7/edm/set-annotations-schema.md

```diff
@@ -0,0 +1,190 @@
+---
+title: "Set schema for out-of-line annotations"
+description: "Set schema for out-of-line annotations"
+author: gathogojr
+ms.author: jogathog
+ms.date: 10/9/2023
+ms.topic: article
+ 
+---
+# Set schema for out-of-line annotations
+**Applies To**: [!INCLUDE[appliesto-odataclient](../../../includes/appliesto-odatalib-v7.md)]
+
+The Edm library lets you [define annotations](/odata/odatalib/edm/define-annotations) and additionally specify the schema that out-of-line annotations should appear in.
```

Comment:
I think giving a simple overview of what an out-of-line annotation is will enrich this doc.

Reply:
@ElizabethOkerio The linked page ([define annotations](/odata/odatalib/edm/define-annotations)) explains both inline and out-of-line annotations so I thought covering the same in this page would be repetitive.

Reply:
@ElizabethOkerio Add a paragraph to introduce annotations (inline & out-of-line)